x/ecocredit/server/migrations.go

```diff
@@ -28,5 +30,11 @@ func (m Migrator) Migrate2to3(ctx sdk.Context) error {
 		return err
 	}
 
+	// add POLYGON to the allowed bridge chain table, as this was a hard coded requirement previously.
+	err := baseStore.AllowedBridgeChainsTable().Insert(ctx, &ecocreditv1.AllowedBridgeChains{ChainName: "POLYGON"})
```

Comment:
We may want to require a governance proposal to add `polygon` but that depends on bridge launch dates and whether we launch before v5.0 goes live. This makes sense for now though. :+1: 

```suggestion
	err := baseStore.AllowedBridgeChainsTable().Insert(ctx, &ecocreditv1.AllowedBridgeChains{ChainName: "polygon"})
```